src/clib/pioc_support.c

```diff
@@ -3821,9 +3821,13 @@ int PIOc_openfile_retry(int iosysid, int *ncidp, int *iotype, const char *filena
             }
             if ((ierr != NC_NOERR) && (file->iotype != PIO_IOTYPE_NETCDF))
             {
-                /* reset file markers for NETCDF on all tasks */
+                /* reset file markers for NETCDF on all IO tasks */
                 file->iotype = PIO_IOTYPE_NETCDF;
 
+                /* FIXME: The changes below to modify the user-specified
+                 * iotype should be made outside the "if (ios->ioproc) {}"
+                 * block, after the switched IO type is broadcast from the
+                 * IO root to all tasks. */
```

Comment:
Can the comment be changed to something like,

"FIXME: The changes below to modify the user-specified iotype needs to be propogated to all tasks/processes"

Reply:
OK, I have updated the FIXME comment.